libafl/src/events/tcp.rs

```diff
@@ -645,8 +645,17 @@ where
             }
 
             #[cfg(feature = "share_objectives")]
```

Comment:
Can we add a runtime flag instead of compile time? Overhead should be minimal and it's easier to use IMHO